### DISCUSSION TITLE
WebDav - 413 Request Entity Too Large

### DISCUSSION THREAD

**Discussion by AudiWolf** [nextcloud/all-in-one#880] [Questions] [answered]
Hello together, 

I want to backup my Mailcow server to my Nextcloud with rclone. I have configured everything via WbDav, but I get the following error for my 16GB big file. -> 413 Request Entity Too Large

Can please anybody help me out? I want to set the limit for WebDav uploads tp 20 GB.

By and thanks in advance,
AudiWolf

**Comment by szaimen** [comment]
Hi, we may allow to adjuet the max upload size via an environemental veriable in thr future

**Comment by szaimen** [comment, marked as answer]
Will be implemented with https://github.com/nextcloud/all-in-one/pull/1028

**Comment by AudiWolf** [comment]
I have tried to adjust the border in the docker compose file but error stays the same.

On my Mailcow-server I fire this statement: rclone sync /mnt/backups nextcloud:/Mailcow-Backup

Errormessage:
 Failed to sync: <html>
<head><title>413 Request Entity Too Large</title></head>
<body>
<center><h1>413 Request Entity Too Large</h1></center>
<hr><center>openresty</center>
</body>
</html>: 413 Request Entity Too Large

**Reply by szaimen** [reply]
This is not released yet.

**Reply by szaimen** [reply]
This is now released with v2.0.0 Beta. Testing and feedback is welcome! See https://github.com/nextcloud/all-in-one#how-to-switch-the-channel
